launch_testing_ros/launch_testing_ros/wait_for_topics.py

```diff
@@ -46,12 +47,14 @@ def method_2():
         print('Given topics are receiving messages !')
         print(wait_for_topics.topics_not_received()) # Should be an empty set
         print(wait_for_topics.topics_received()) # Should be {'topic_1', 'topic_2'}
+        print(wait_for_topics.messages_received('topic_1')) # Should be [message_1, ...]
         wait_for_topics.shutdown()
     """
 
-    def __init__(self, topic_tuples, timeout=5.0):
+    def __init__(self, topic_tuples, timeout=5.0, max_number_of_messages=100):
```

Comment:
I'd keep the default size small, say 5 or 10 and then the users can decide if they want longer queue sizes.

Reply:
Thank you for the suggestion. I fixed it in commit 0f4753521b74f8505b17a98b496f7bfee60279a0.
By the way, for such small suggestions, it can be more efficient to directly [suggest a code change through the GitHub interface](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/incorporating-feedback-in-your-pull-request).